### PR TITLE
Test implement

### PR DESCRIPTION
**Pull request by lrojas870**
Implement test:

- implement test to viewModel

### CODE REVIEW

File: app/src/androidTest/java/com/androidavanzado/covid19stadistics/ExampleInstrumentedTest.kt [hunk 1]
```diff
@@ -15,10 +22,31 @@ import org.junit.Assert.*
...
+    }
+
     @Test
-    fun useAppContext() {
-        // Context of the app under test.
-        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
-        assertEquals("com.androidavanzado.covid19stadistics", appContext.packageName)
+    fun validateTextTitleTv(){
+        Thread.sleep(8000)
+        val title = DateTextFormat().invoke(dateYesteday)
```
Comment (danponce) [inline app/src/androidTest/java/com/androidavanzado/covid19stadistics/ExampleInstrumentedTest.kt:44]: I'm guessing that this numbers in **confirmedCases** and **deathCases** are related especifically to this date and not yesterday date. Shouldn't be an specific date instead?

File: app/src/test/java/com/androidavanzado/covid19stadistics/ui/StadisticsViewModelTest.kt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        testCoroutineRule.runBlockingTest {
+
+            val dateYesterday = DateYesterday().invoke()
+            val result = StadisticsModelFactory().getStadisticsDataSuccess()
+            val stadisticsApi = StadisticsModelFactory().getStadisticsObjet()
+
+            Mockito.`when`(repositoryApi.getData(dateYesterday)).thenReturn(result)
+
+            stadisticsViewModel.getData(dateYesterday)
+            Thread.sleep(5000)
```
Comment (danponce) [inline app/src/test/java/com/androidavanzado/covid19stadistics/ui/StadisticsViewModelTest.kt:57]: Can you add comments of why you have to add a Thread.sleep() ?

File: app/src/test/java/com/androidavanzado/covid19stadistics/util/DateTomorrow.kt [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package com.androidavanzado.covid19stadistics.util
+
+import android.annotation.SuppressLint
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+
+@SuppressLint("SimpleDateFormat")
+class DateTomorrow {
```
Comment (danponce) [inline app/src/test/java/com/androidavanzado/covid19stadistics/util/DateTomorrow.kt]: This class I think it's going to be used only in StadisticsViewModelTest so why not changing it's name to StadisticsViewModelTestDateUtils ?

Comment (lrojas870) [reply app/src/test/java/com/androidavanzado/covid19stadistics/util/DateTomorrow.kt]: done, thanks

File: app/src/test/java/com/androidavanzado/covid19stadistics/ui/StadisticsViewModelTest.kt (hunk 2 above)
Comment (lrojas870) [reply app/src/test/java/com/androidavanzado/covid19stadistics/ui/StadisticsViewModelTest.kt:57]: done, Thread.sleep () was commented in the tests

File: app/src/androidTest/java/com/androidavanzado/covid19stadistics/ExampleInstrumentedTest.kt (hunk 1 above)
Comment (lrojas870) [reply app/src/androidTest/java/com/androidavanzado/covid19stadistics/ExampleInstrumentedTest.kt:44]: done, thanks

**Pull request merged**